plugins/panels/similarity_search/run_manager.py

```
@@ -0,0 +1,175 @@
+
+logger = logging.getLogger(__name__)
+
+
+class RunManager:
+    """Manager class for persisting and retrieving similarity search runs."""
+
+    _RUN_PREFIX = "run:"
+    _OPID_PREFIX = "opid:"
+
+    # Fields too large for listing — only needed by get_run / apply_run
+    _HEAVY_FIELDS = {"result_ids", "result_view", "source_view"}
```

Ruff (0.15.5)

[warning] Mutable default value for class attribute

(RUF012)